src/map/clif.cpp

```diff
@@ -17674,6 +17674,13 @@ void clif_parse_configuration( int32 fd, map_session_data* sd ){
 
 			sd->hd->homunculus.autofeed = flag;
 			break;
+		case CONFIG_DISABLE_SHOWCOSTUMES:
+			if(flag == true)
+				sd->status.disable_showcostumes = false;
+			else
+				sd->status.disable_showcostumes = true;
```

Comment:
following your logic, we could write that
```suggestion
			sd->status.disable_showcostumes = !flag;
```
eventually

Reply:
[Ragexe2025-05-05.zip](https://github.com/user-attachments/files/20237233/Ragexe2025-05-05.zip)
Ragexe2025-05-05